src/nunit.gui.core/Views/IXmlView.cs

```diff
@@ -33,7 +31,7 @@ public interface IXmlView : IView
         bool Visible { get; set; }
         string Header { get; set; }
         IViewElement XmlPanel { get; }
-        IControlElement<RichTextBox> XmlTextBox { get; }
+        IXmlTextBoxView XmlTextBox { get; }
```

Comment:
This solves the problem. I'll need to come back and integrate it with the the controls in UiKit.Elements in a subsequent commit.